functions.opengraph.php

```diff
@@ -261,22 +261,20 @@ function jetpack_og_tags() {
 }
 
 function jetpack_og_get_image( $width = 200, $height = 200, $max_images = 4 ) { // Facebook requires thumbnails to be a minimum of 200x200
-	$image = '';
+	$image = array();
 
 	if ( is_singular() && ! is_home() ) {
 		global $post;
```

Comment:
@georgestephanis I comitted fe54292a4 for the `$post` change. Since it's not directly related to this issue/PR, I'm fine with that change being reverted and/or committed to a new branch.